src/components/ui/radio-button-group.module.css

```diff
@@ -17,6 +17,11 @@
   padding: 0.5rem 0.5rem;
   user-select: none;
 
+  &.bare {
+    border: 0;
```

Comment:
We need to be careful with nulling borders if other variants have them - if the buttons ever get mixed, the button's sizes will be unequal. a `transparent` border would maintain layout.